packages/next/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -598,6 +617,9 @@ export class TraceEntryPointsPlugin implements webpack5.WebpackPluginInstance {
           if (!res) {
             throw new Error(`failed to resolve ${request} from ${parent}`)
           }
+          if (res.includes('?')) {
+            console.log('wtf?', res)
```

Comment:
Lets use a better error message here 😅

Reply:
Yeah this was from debugging, good catch 😅